src/features/groups/components/GroupCardList/GroupCard/GroupCardContent/GroupCardDropdownMenuTriggerButton/GroupCardDropdownMenuContent/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { PenIcon } from '@/assets/images/icons';
+import {
+  DropdownMenuButton,
+  DropdownMenuButtonIcon,
+  DropdownMenuButtonText,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  Icon,
+} from '@/components/ui';
+
+import { DeleteGroupDialogTriggerButton } from './DeleteGroupDialogTriggerButton';
+
+interface IGroupCardDropdownMenuContentProps {
+  /**
+   * Function to handle the rename button click.
+   */
+  handleRenameClick: () => void;
+}
+
+export const GroupCardDropdownMenuContent = ({
+  handleRenameClick,
+}: IGroupCardDropdownMenuContentProps) => {
+  /**
+   * The function that is called when the dropdown menu item is selected.
+   * e.preventDefault() prevents the dropdown menu from closing when selecting that item.
+   * This is necessary because closing dropdown menu also closes the dialog unintentionally.
+   * @see {@link https://www.radix-ui.com/primitives/docs/components/dropdown-menu#item}
+   *
+   * @param e The event object
```

Comment:
This is a minor thing, though; here is the official way of formatting the `@param` tag. It is good to be aware of.
```ts
/**
 * @param e - The event object
 */
```

https://tsdoc.org/pages/tags/param/